lms/templates/courseware/progress.html

```diff
@@ -173,7 +177,7 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
                 ${ section['display_name'] }
                 %if total > 0 or earned > 0:
                   <span class="sr">
-                    ${_("{earned:.3n} of {total:.3n} possible points").format( earned = float(earned), total = float(total) )}
+                    ${_("{earned} of {total} possible points").format(earned='{:.3n}'.format(float(earned)), total='{:.3n}'.format(float(total)))}
```

Comment:
Is it necessary to coerce `earned` and `total` to float? Shouldn't the formatting do that already?